hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/SerializationUtil.java

```diff
@@ -54,7 +54,7 @@
 
 public final class SerializationUtil {
 
-    static final PartitioningStrategy EMPTY_PARTITIONING_STRATEGY = new EmptyPartitioningStrategy();
+    public static final PartitioningStrategy EMPTY_PARTITIONING_STRATEGY = new EmptyPartitioningStrategy();
```

Comment:
This change will not be needed if we remove `AllFieldsRowPartitioner`, which, in turn, should not be needed for this PR either.

Reply:
Addressed in https://github.com/hazelcast/hazelcast/pull/18013/commits/cd3befd88ecdad90b2cc8fe3facf30290dcbedc4